src/Compilers/CSharp/Test/Emit2/Semantics/RefReadonlyParameterTests.cs

```diff
@@ -44,17 +44,7 @@ private static void VerifyRequiresLocationAttributeSynthesized(ModuleSymbol modu
 
         if (attributes)
         {
-            if (parameter.ContainingModule is SourceModuleSymbol)
-            {
-                Assert.Empty(parameter.GetAttributes());
-            }
-            else
-            {
-                var attribute = Assert.Single(parameter.GetAttributes());
-                Assert.Equal("System.Runtime.CompilerServices.RequiresLocationAttribute", attribute.AttributeClass.ToTestDisplayString());
-                Assert.Empty(attribute.ConstructorArguments);
-                Assert.Empty(attribute.NamedArguments);
-            }
+            Assert.Empty(parameter.GetAttributes());
```

Comment:
Unfortunately, it means the presence of the attribute cannot be verified using compiler APIs anymore, I think. But it's still verified in the sense that it makes `PEParameterSymbol.RefKind` decoded correctly as `RefReadonlyParameter` if present.